drivers/sqlite-driver/src/main/kotlin/com/squareup/sqldelight/sqlite/driver/JdbcSqliteDriver.kt

```diff
@@ -18,131 +11,16 @@ class JdbcSqliteDriver constructor(
    */
   url: String,
   properties: Properties = Properties()
-) : SqlDriver {
+) : JdbcDriver() {
```

Comment:
Do we even want this subclass anymore? What about having the `JdbcDriver` take a `DriverManager` and just deprecate this whole subclass? You could probably use interface delegation if we did that
```kotlin
class JdbcSqliteDriver(stuff) : SqlDriver by JdbcDriver(DriverManager.getConnection(url, properties))
```

Reply:
i'm predicting we'll need to be more flexible with how connections are retrieved for mysql. I'm open to getting rid of the subclass, but I do think JdbcDriver needs to be open and I kept JdbcSqliteDriver around for binary compatibility - if we dont think its important im fine dropping it

Reply:
No maintaining it is of the utmost importance. I'm just figuring out if we can make the JDBC driver useful on its own rather than requiring DB-specific subtypes. How do other libraries handle this?

Reply:
I think the problem is that it's container specific, so misk will need it own driver. i dont know of other containers. its too early for me to really know and i dont think other libraries (like hibernate) work at this abstraction

Reply:
Checked some side projects of mine from long ago. They all use `javax.sql.DataSource` as the abstraction for a connection pool.

Reply:
https://github.com/cashapp/misk/blob/master/misk-hibernate/src/main/kotlin/misk/jdbc/DataSourceService.kt

looks consistent

once i get things legitimately working i think we can revisit the jdbc driver api and simplify things